### PR TITLE
bmips: add support for TP Link TD-W8968 V3, TD-W8968 V4 and TD-W8960N V5

### PR DESCRIPTION
**Pull request by Noltari**
Supersedes https://github.com/openwrt/openwrt/pull/15882

Fixes author and adds additional models to ALT0/ALT1.

### CODE REVIEW

File: target/linux/bmips/dts/bcm6318-tp-link-td-w8968-v3.dts [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+		/* EPHY3 Act */
+		reg = <7>;
+		brcm,hardware-controlled;
+		brcm,link-signal-sources = <7>;
+	};
+
+	led@8 {
+		reg = <8>;
+		active-low;
+		label = "green:internet";
```
Comment (musashino205) [inline target/linux/bmips/dts/bcm6318-tp-link-td-w8968-v3.dts:169]: suggestion:

```
color = <LED_COLOR_ID_GREEN>;
function = LED_FUNCTION_WAN_ONLINE;
```

File: target/linux/bmips/dts/bcm6318-tp-link-td-w8968-v3.dts [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+		reg = <13>;
+		active-low;
+		function = LED_FUNCTION_USB;
+		color = <LED_COLOR_ID_GREEN>;
+	};
+	
+	led@16 {
+		reg = <16>;
+		active-low;
+		label = "green:wifi";
```
Comment (musashino205) [inline target/linux/bmips/dts/bcm6318-tp-link-td-w8968-v3.dts:188]:
```
color = <LED_COLOR_ID_GREEN>;
function = LED_FUNCTION_WLAN;
```

File: target/linux/bmips/dts/bcm6318-tp-link-td-w8968-v3.dts [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+	aliases {
+		led-boot = &led_power_green;
+		led-failsafe = &led_power_green;
+		led-running = &led_power_green;
+		led-upgrade = &led_power_green;
+	};
+	
+	keys {
+		compatible = "gpio-keys-polled";
+		poll-interval = <100>;
```
Comment (musashino205) [inline target/linux/bmips/dts/bcm6318-tp-link-td-w8968-v3.dts:18]: Isn't `gpio-keys` available?

Comment (Noltari) [reply target/linux/bmips/dts/bcm6318-tp-link-td-w8968-v3.dts:18]: > Isn't `gpio-keys` available?

Nope, external interrupts aren't supported yet.

File: target/linux/bmips/dts/bcm6318-tp-link-td-w8968-v3.dts (hunk 1 above)
Comment (Noltari) [reply target/linux/bmips/dts/bcm6318-tp-link-td-w8968-v3.dts]: Fixed, thanks!

File: target/linux/bmips/dts/bcm6318-tp-link-td-w8968-v3.dts (hunk 2 above)
Comment (Noltari) [reply target/linux/bmips/dts/bcm6318-tp-link-td-w8968-v3.dts]: Fixed, thanks!

**Pull request merged**